packages/contracts/test/newsroom/ACL.ts

```diff
@@ -62,9 +62,11 @@ contract("ACL", (accounts: string[]) => {
       const event = findEvent(tx, events.ACL_ROLE_ADDED);
 
       expect(event).to.not.be.null();
-      expect(event.args.granter).to.be.equal(defaultAccount);
-      expect(event.args.grantee).to.be.equal(accounts[1]);
-      expect(event.args.role).to.be.equal(ACL_TEST_ROLE);
+      // tslint:disable no-non-null-assertion
```

Comment:
we should either disable this check in tslint.json or put the 3 lines below inside an `if (event)` and remove the !'s

Reply:
I've started a poll. Gonna merge this one, and if everyone's in favour, I'll remove the lint rule